### ISSUE TITLE
JS refactory

### ISSUE THREAD

**Issue by robsongajunior** [es6rocks/harmonic-theme-jsrocks#33]
It will be really coll rewrite JavaScript to ES2015.

**Comment by UltCombo** [comment]
Yeah, the front-end preset that I was supposed to land on 20xx would be helpful here.

**Comment by UltCombo** [comment]
Will probably have time to land the 20xx front-end preset in the Sunday, I'll work the full day tomorrow.
